src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -5153,6 +5153,12 @@ void func_8083AF44(PlayState* play, Player* this, s32 magicSpell) {
     func_80835DE4(play, this, func_808507F4, 0);
 
     this->unk_84F = magicSpell - 3;
+
+    //! @bug `MAGIC_CONSUME_WAIT_PREVIEW` is not guaranteed to succeed. Ideally, the return value of
+    //! `Magic_RequestChange` should be checked before allowing the process of using a spell to continue.
+    //! If the magic state change request fails, `magicTarget` will never be set correctly.
+    //! When `MAGIC_STATE_CONSUME_SETUP` is set in `func_808507F4`, magic will eventually be consumed to a stale target
+    //! value. If the stale target value is higher than the current magic value, it will be drained to zero.
```

Comment:
and if it's lower it will just drain to the stale target? (or what really happens in that case?)

Reply:
It will just drain to target yeah. I only mentioned the higher than case cause its less intuitive to know what it will do

Reply:
I think it's nice to add it for completeness so the curious reader won't have to dredge through this stuff again to find the alternative outcome

Reply:
`magic will eventually be consumed to a stale target value` before that sentence is meant to convey this. but if you have a suggestion for rewording, lmk

Reply:
I think my confusion stemmed from one using "consumed" and the other "drained" maybe, I think making the wording consistent between both should clear it up

Reply:
sure, changed